output/forms.eslint.txt

```diff
@@ -13,11 +13,19 @@ The react/require-extension rule is deprecated. Please use the import/extensions
 /home/travis/build/Talend/ui/packages/forms/src/UIForm/fields/Radios/Radios.component.js
   24:7  error  Form controls using a label to identify them must be programmatically associated with the control using htmlFor  jsx-a11y/label-has-for
 
+/home/travis/build/Talend/ui/packages/forms/src/UIForm/UIForm.component.test.js
```

Comment:
i can't fix it, it's due to travis's modifications

Reply:
Hmm yes that's weird, eslint is wrong